app/assets/javascripts/services/stories.js

```diff
@@ -0,0 +1,9 @@
+import httpService from './httpService';
+
+class ProjectStoriesService {
+  async fetchStory(story) {
+    return httpService.get(`/projects/${story.projectId}/stories/${story.id}`);
+  }
+}
+
+export default new ProjectStoriesService();
```

Comment:
We can use a plain object here instead of a class, since we never use the class, only the instance:

```suggestion
const ProjectStoriesService = {
  async fetchStory(story) {
    return httpService.get(`/projects/${story.projectId}/stories/${story.id}`);
  }
}

export default ProjectStoriesService;
```